app/models/item.rb

```diff
@@ -7,4 +7,12 @@ class Item < ActiveRecord::Base
   validates :region, presence: true
   validates :eve_type, presence: true
   validates :info_date, presence: true
+
+  def self.search(search)
+    if search
+      where('name LIKE ?', "%#{search}%")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.